R/span.R

```diff
@@ -0,0 +1,101 @@
+#' Calculate time span between dates
+#'
+#' @param data The input data frame or linelist
+#' @param target_column A string used to specify the name of the date column of
+#'    interest. The values in this columns should be of type 'Date' in ISO
```

Comment:
Can `target_column` be a vector of multiple columns? If so, consider renaming the param `target_columns` if not consider changing columns to singular.

Reply:
fixed in commit `71ff418`